README.md

```diff
@@ -227,3 +227,12 @@ about the system requirements to support this mode of operation can be found [he
 
 EOF
 ```
+
+## Limitations
+### RDMA workloads utilizing RDMA Connection Manager (CM)
+
+When RDMA isolation is enabled, pods creation/deleteion can't be in parallel.
+This due to the face that moveing RDMA device to namespace and rebinding
```

Comment:
deleteion/deletion
face/fact
moveing/moving
diffrent/different
This due/This is due

Reply:
nit (rephrasing): can't be in parallel -> cannot <occur|happen> in parallel